### PR TITLE
DCOS-8058: Add client side health check validation

### PR DESCRIPTION
**Pull request by orlandohohmeier**

### CODE REVIEW

**Comment by wavesoft** [comment]
LGTM 👍

File: src/js/schemas/service-schema/HealthChecks.js [hunk 1]
```diff
@@ -65,46 +67,163 @@ let HealthChecks = {
...
+            },
+            externalValidator: function ({healthChecks}, definition) {
+              const index = FormUtil.getPropIndex(definition.name);
+              const {[definition.name]: command} = healthChecks;
+              const {[`healthChecks[${index}].protocol`]: protocol}
+                = healthChecks;
+
+              if (protocol === 'COMMAND' && !ValidatorUtil.isDefined(command)) {
+                definition.showError =
+                  'Command must not be empty.';
```
Comment (rcorral) [inline src/js/schemas/service-schema/HealthChecks.js]: Does this need to be in a new line?

**Comment by rcorral** [comment]
Aside from that small comment, it looks good!

File: src/js/schemas/service-schema/HealthChecks.js (hunk 1 above)
Comment (orlandohohmeier) [inline src/js/schemas/service-schema/HealthChecks.js]: Fixed

**Pull request merged**